keystore/filesystem/server_keystore_test.go

```diff
@@ -768,3 +976,58 @@ func testHistoricalKeyAccess(storage Storage, t *testing.T) {
 		}
 	}
 }
+
+// BenchmarkHistoricalPathsSerialization compare simple way of serialization []string into []byte with strings.[Split|Join] and msgpack
+// Example of benchmarks to see comparison how msgpack faster simple join/split operations with strings
+// goos: linux
+// goarch: amd64
+// pkg: github.com/cossacklabs/acra/keystore/filesystem
+// cpu: Intel(R) Core(TM) i7-10750H CPU @ 2.60GHz
```

Comment:
Did you lock CPU frequency at this specific (or some other) value when benchmarking?
BTW, to get more precise results, instead of closing other apps, sending them SIGSTOP will also do the trick :+1:

Reply:
for what we need here such precise benchmarks?  there are 10^10 difference between approaches. you think such improvements may change general picture of that and we should spent time for that?)

Reply:
it's just simple benchmark to show other developers who will read code and think: why you used this serialization algo instead of some more simple from std. and to allow everyone reproduce by one command on their machine to be sure

Reply:
Just saying what comes in mind. Here we clearly see that `msgpack` is much better, that's enough.

BTW, if you wanted to try something like this, I got a shell function to set min/max frequencies, at least it worked for some Intel laptop CPU
```
set-cpu-freq-min-max() {
	[[ $1 > 0 && $2 > 0 ]] || { echo "$0 <min_freq_KHz> <max_freq_KHz>"; return }
	echo $1 | sudo tee /sys/devices/system/cpu/cpu*/cpufreq/scaling_min_freq
	echo $2 | sudo tee /sys/devices/system/cpu/cpu*/cpufreq/scaling_max_freq
}
```